.golangci.yml

```diff
@@ -117,61 +113,91 @@ linters:
 #   - wsl
 #   - zerologlint
 
-linters-settings:
-  depguard:
+  settings:
+    depguard:
+      rules:
+        prevent_json_package_use:
+          list-mode: lax
+          files:
+            - $all
+          allow:
+            - github.com/thrasher-corp/gocryptotrader/encoding/json
+          deny:
+            - pkg: encoding/json
+              desc: Please use the custom JSON package github.com/thrasher-corp/gocryptotrader/encoding/json instead of encoding/json to allow sonic/json switching.
+    goconst:
+      min-occurrences: 6
+    gocritic:
+      disabled-checks:
+        - wrapperFunc
+        - importShadow
```

Comment:
Might be able to drop this soon `Detects when imported package names shadowed in the assignments.` which we do in review anyway.

Reply:
I agree, upon looking into it I stumbled upon an issue which was filed here: https://github.com/golangci/golangci-lint/issues/5672

A PR has been opened for it and undergoing review